config/database.yml

```diff
@@ -24,9 +24,9 @@ default: &default
   checkout_timeout: 2
   variables:
       statement_timeout: <%= ENV['STATEMENT_TIMEOUT'] || 2500 %>
-      
 development:
   <<: *default
+  url: <%= ENV.fetch('DATABASE_URL') %>
```

Comment:
This uses .fetch which [throws a KeyError](Your board will ship in 3-4 weeks.
) if no default is provided and the value doesn't exist. Will this have unexpected side effects for non-docker users?

Reply:
Good catch 👍 . As part of testing this PR, we need to do a clean install for non-docker users.